main/src/library/MutList.flix

```diff
@@ -174,39 +447,103 @@ namespace MutList {
     ///
     /// If the given index `i` exceeds the length of the mutable list, no element is removed.
     ///
-    pub def remove(v: mut MutList[a], i: Int32): Unit = ???
+    pub def remove(i: Int32, v: mut MutList[a]): Unit & Impure =
+        let MutList(a, l) = v;
+        let len = deref l;
+        if (i < len) {
+            a := Array.init(j -> if (j < i) (deref a)[i] else (deref a)[i + 1], len - 1);
+            l := len - 1
+        }
+        else
+            ()
 
     ///
     /// Inserts all the elements in the given mutable list `v2` into the mutable list `v1`.
     ///
-    pub def append(v1: mut MutList[a], v2: MutList[a]): Unit = ???
+    pub def append(v1: MutList[a], v2: MutList[a]): Unit & Impure =
```

Comment:
Put the `mut` modifier. I believe it goes on `v1`. 

Also, do a quick check to see if it is missing from any other places.